src/diffusers/pipelines/cogvideo/pipeline_cogvideox_image2video.py

```diff
@@ -666,11 +671,6 @@ def __call__(
             `tuple`. When returning a tuple, the first element is a list with the generated images.
         """
 
-        if num_frames > 49:
```

Comment:
We need to remove the frame restriction since the newer models can generate at higher frame resolutions. 49 frames will still be the default for 1.0 models because we determine that by the transformer config params